Cargo.toml

```diff
@@ -35,6 +35,7 @@ cortex-m-semihosting = "0.3.2"
 void = { version = "1.0.2", default-features = false }
 cast = { version = "0.2.2", default-features = false }
 nb = "0.1.2"
+stm32-usbd = { version = "0.5.0", features = ["ram_access_2x16"], optional = true }
```

Comment:
Not sure if the dependency needs to be optional. Yes, it's a heavy one, but that's only relevant on the first compile, as embedded applications will use LTO to remove unused stuff anyway. Maybe I'm wrong.

Reply:
It's just a dependency tree (==build time) optimization, it doesn't affect the code size.